public/app/percona/pmm-dump/PMMDump.tsx

```diff
@@ -0,0 +1,328 @@
+  }, [loadData]);
+
+  const closeEditModal = (saved = false) => {
+    setIsSendToSupportModalOpened(false);
+    setSelectedDump(null);
+  };
+
+  const getActions = useCallback(
+    (row: Row<PMMDumpServices>): Action[] => [
+      {
+        content: (
+          <HorizontalGroup spacing="sm">
+            <Icon name="trash-alt" />
+            <span className={styles.actionItemTxtSpan}>{Messages.services.actions.delete}</span>
+          </HorizontalGroup>
+        ),
+        action: () => {
+          onDelete(row.original);
+        },
+      },
+      {
+        content: (
+          <HorizontalGroup spacing="sm">
+            <Icon name="arrow-right" />
+            <span className={styles.actionItemTxtSpan}>{Messages.services.actions.sendToSupport}</span>
+          </HorizontalGroup>
+        ),
+        action: () => {
+          setSelectedDumpIds([row.original.dump_id]);
+          setIsSendToSupportModalOpened(true);
+        },
+      },
+      {
+        content: (
+          <HorizontalGroup spacing="sm">
+            <Icon name="eye" />
+            <span className={styles.actionItemTxtSpan} onClick={() => onLogClick(row.original)}>
+              {Messages.services.actions.viewLogs}
+            </span>
+          </HorizontalGroup>
+        ),
+        action: () => {
+          // locationService.push(`/d/pmm-qan/pmm-query-analytics?var-service_name=${row.original.serviceName}`);
+        },
+      },
+    ],
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [styles.actionItemTxtSpan]
+  );
+
+  const onDelete = (value?: PMMDumpServices) => {
+    if (value) {
+      appEvents.publish(
+        new ShowConfirmModalEvent({
+          title: 'Delete',
+          text: 'Are you sure you want to delete this PMM dump?',
+          yesText: 'Delete',
+          icon: 'trash-alt',
+          onConfirm: () => {
+            dispatch(deletePmmDumpAction([value.dump_id]));
+          },
+        })
+      );
+    } else if (selected.length > 0) {
+      appEvents.publish(
+        new ShowConfirmModalEvent({
+          title: 'Delete',
+          text: 'Are you sure you want to delete the selected dumps?',
+          yesText: 'Delete',
+          icon: 'trash-alt',
+          onConfirm: () => {
+            const dump_ids = selected.map((item) => item.original.dump_id);
+            dispatch(deletePmmDumpAction(dump_ids));
+          },
+        })
+      );
+    }
+    loadData();
+  };
+
+  const columns = useMemo(
+    (): Array<ExtendedColumn<PMMDumpServices>> => [
+      {
+        Header: Messages.services.columns.id,
+        id: 'dump_id',
+        accessor: 'dump_id',
+        hidden: true,
+        type: FilterFieldTypes.TEXT,
+      },
+      {
+        Header: Messages.services.columns.status,
+        accessor: 'status',
+        Cell: ({ value }: { value: DumpStatus }) => {
+          return <Badge text={DumpStatusText[value]} color={DumpStatusColor[value as DumpStatus] as BadgeColor} />;
```

Comment:
Use implicit return when possible (where it makes sense).

Reply:
true